app/javascript/src/BackendApis/components/BackendSelect.jsx

```diff
@@ -15,10 +15,11 @@ type Props = {
   backends: Backend[],
   onCreateNewBackend: () => void,
   error?: string,
-  onSelect: (Backend | null) => void
+  searchPlaceholder?: string,
+  onSelect: (Backend | null) => void,
```

Comment:
Now it is the same change, just 1 line after 😂

Reply:
Well, I'll take this into account for the next time if that's ok :joy: